skt/executable.py

```diff
@@ -123,7 +125,7 @@ def cmd_merge(cfg):
         idx = 0
         for mb in cfg.get('merge_ref'):
             save_state(cfg, {'meregerepo_%02d' % idx: mb[0],
-                             'mergehead_%02d' % idx: head})
+                             'mergehead_%02d' % idx: bhead})
```

Comment:
Hmm, could you please make this a separate commit and elaborate a little bit on what's going on? Thanks!